VoucherifyRequest.php

```diff
@@ -88,6 +88,10 @@ protected function apiRequest($method, $endpoint, $params, $data) {
             throw new ClientException($error);
         } // Invalid status code
         else if ($statusCode >= 400) {
+            $resultDecoded = json_decode($result, true);
```

Comment:
please leave current error response format - we will update ClientException in next iteration to match API error format { code, message, detail }